lib/index.js

```diff
@@ -199,6 +245,25 @@ conversions["USVString"] = function (V) {
     return U.join('');
 };
 
+conversions["object"] = function (V, opts) {
```

Comment:
Omit `opts` when it's not needed. Lint should be catching this... maybe not set up on this repo yet.